pytest_pootle/fixtures/models/translation_project.py

```diff
@@ -79,3 +81,24 @@ def afrikaans_vfolder_test(afrikaans, vfolder_test):
 def templates_tutorial(templates, tutorial):
     """Require Template Tutorial."""
     return _require_tp(templates, tutorial)
+
+
+def project_checkers():
```

Comment:
nit: as this is not a fixture, could we perhaps move it pytest_pootle.utils and maybe make it get_project_checkers